### DISCUSSION TITLE
Docker install not completing

### DISCUSSION THREAD

**Discussion by duncanjairos** [netbox-community/netbox-docker#751] [Getting Help]
Hi 

Can anyone assist with some info. i have tried cloning this and following the steps to install but seems like the installation always stops at the same spot, not sure what im missing. I have tried different Linux instances and distros and still same issue

###

netbox_1               | Running the startup script /opt/netbox/startup_scripts/420_providers.py
netbox_1               |   Running the startup script /opt/netbox/startup_scripts/440_circuit_types.py
netbox_1               |   Running the startup script /opt/netbox/startup_scripts/450_circuits.py
netbox_1               |  Initialisation is done.
netbox_1               |  Waiting for control socket to be created... (1/10)
netbox_1               | 2022/04/25 12:42:50 [warn] 7#7 Unit is running unprivileged, then it cannot use arbitrary user and group.
netbox_1               | 2022/04/25 12:42:50 [info] 7#7 unit started
netbox_1               | 2022/04/25 12:42:50 [info] 20#20 discovery started
netbox_1               | 2022/04/25 12:42:50 [notice] 20#20 module: python 3.9.5 "/usr/lib/unit/modules/python3.unit.so"
netbox_1               | 2022/04/25 12:42:50 [info] 7#7 controller started
netbox_1               | 2022/04/25 12:42:50 [notice] 7#7 process 20 exited with code 0
netbox_1               | 2022/04/25 12:42:50 [info] 22#22 router started
netbox_1               | 2022/04/25 12:42:50 [info] 22#22 OpenSSL 1.1.1n  15 Mar 2022, 101010ef
netbox_1               |  Applying configuration from /etc/unit/nginx-unit.json
netbox_1               | 2022/04/25 12:42:51 [info] 26#26 "netbox" application started
netbox_1               |  Unit configuration loaded successfully
netbox_1               | 2022/04/25 12:42:53 [notice] 7#7 process 18 exited with code 0

###

**Comment by kkthxbye-code** [comment]
That is the expected log output of a successful start.

This is a log from at docker-compose up I just did:

```
netbox-docker-netbox-1               | 2022/04/25 14:34:32 [warn] 9#9 Unit is running unprivileged, then it cannot use arbitrary user and group.
netbox-docker-netbox-1               | 2022/04/25 14:34:32 [info] 9#9 unit started
netbox-docker-netbox-1               | 2022/04/25 14:34:32 [info] 18#18 discovery started
netbox-docker-netbox-1               | 2022/04/25 14:34:32 [notice] 18#18 module: python 3.9.2 "/usr/lib/unit/modules/python3.9.unit.so"
netbox-docker-netbox-1               | 2022/04/25 14:34:32 [info] 9#9 controller started
netbox-docker-netbox-1               | 2022/04/25 14:34:32 [info] 20#20 router started
netbox-docker-netbox-1               | 2022/04/25 14:34:32 [notice] 9#9 process 18 exited with code 0
netbox-docker-netbox-1               | 2022/04/25 14:34:32 [info] 20#20 OpenSSL 1.1.1n  15 Mar 2022, 101010ef
netbox-docker-netbox-1               | ⚙️ Applying configuration from /etc/unit/nginx-unit.json
netbox-docker-netbox-1               | 2022/04/25 14:34:33 [info] 24#24 "netbox" prototype started
netbox-docker-netbox-1               | 2022/04/25 14:34:33 [info] 25#25 "netbox" application started
netbox-docker-netbox-1               | ✅ Unit configuration loaded successfully
netbox-docker-netbox-1               | 2022/04/25 14:34:34 [notice] 9#9 process 16 exited with code 0
```
